### PR TITLE
Add support for Multilevel Switch Notification

### PR DESCRIPTION
**Pull request by reubenbijl**
https://github.com/home-assistant-libs/zwave-js-server-python/issues/396

### CODE REVIEW

**Comment by MartinHjelmare** [comment]
This requires node-zwave-js version 9, right? This PR then probably should be merged after we merge https://github.com/home-assistant-libs/zwave-js-server-python/pull/381.

**Comment by reubenbijl** [comment]
Yes, correct, requires z-wave-js version 9.

File: zwave_js_server/model/notification.py
Comment (raman325) [inline zwave_js_server/model/notification.py:192]:
```suggestion
class MultilevelSwitchNotificationArgsDataType(TypedDict, total=False):
    """Represent args for a Multi Level Switch CC notification event data dict type."""

    event_type: int  # required
    direction: str
```
I believe `direction` is optional here right?

Comment (reubenbijl) [reply zwave_js_server/model/notification.py:192]: Yes, if the event type is a Multilevel Switch Stop event, then there is not direction associated with it.

**Comment by raman325** [comment]
please run black on the file to remove the extra whitespaces

File: zwave_js_server/model/notification.py
Comment (raman325) [inline zwave_js_server/model/notification.py:220]: We should create an `IntEnum` for `MultilevelSwitchCommand` in `zwave_js_server.const.command_class.multilevel_switch` and then return the enum value here instead of the raw int

Comment (reubenbijl) [reply zwave_js_server/model/notification.py:220]: Do you mean for the event type?
ie., new enum in         return CommandClass.SWITCH_MULTILEVEL with something like this:

class MultilevelSwitchCommand(IntEnum):
	START_LEVEL = 4
	STOP = 5

Or for the Command classId?
return CommandClass.SWITCH_MULTILEVEL

Comment (reubenbijl) [reply zwave_js_server/model/notification.py]: @raman325  see commit, I think this is what you meant.

File: zwave_js_server/const/command_class/multilevel_switch.py
Comment (raman325) [inline zwave_js_server/const/command_class/multilevel_switch.py:14]: not necessary for this PR but we may as well define all the possible enums here

**Review by raman325 — APPROVED** [review]
this can be merged as soon as tests pass and after https://github.com/home-assistant-libs/zwave-js-server-python/pull/381 gets merged. This will happen soon, just waiting a bit to make sure there are no issues with the latest HA release before we bump the schema in that PR

**Comment by raman325** [comment]
please add tests for this new notification type

**Review by raman325 — CHANGES_REQUESTED** [review]
please add tests to improve coverage

**Review by raman325 — CHANGES_REQUESTED** [review]
one more run of black and we **should** be good

**Review by raman325 — APPROVED** [review]
great, just need to merge the other PR!

**Pull request merged**
